docs/MD.md

```diff
@@ -1,29 +1,129 @@
 # Maskwacîs Dictionary
 
-## General Notes
-
-The TSV version of the Maskwacîs dictionary in the ALTLab repo also has semantic classifications, using the Rapid Words ontology.
+Notes on the Maskwacîs dictionary database.
 
 ## SRO
 
-* Inconsistent marking of vowel length.
+A transcription of the headword in Standard Roman Orthography (SRO).
+
+* Occasionally includes punctuation, **ex:** _ah?_ 'What? What did you say?'.
+* Vowel length is indicated with double vowels rather than macrons or circumflexes, **ex:** `aa`.
```

Comment:
Nope, as far as I know. Rather, long vowels lack diacritical marking.

Reply:
I found four instances of doubled vowels in the data:

* `ikatestaa	ᐃᑲᑌᐢᑕᐊ	vc	Put it aside somewhere else.`
* `sihtoskaawew	ᓯᐦᑐᐢᑲᐊᐁᐧᐤ	vp	He supports him.`
* `kiipahikan	ᑭᐃᐸᐦᐃᑲᐣ	n	A lid, cork or stopper.`
* `kiipiskah	ᑭᐃᐱᐢᑲᐦ	vc	Block the way.`
* `kiisapiskisam	ᑭᐃᓴᐱᐢᑭᓴᒼ	v	He heats a metal object.`